src/schema/train-booking.json

```diff
@@ -180,8 +179,24 @@
             "type": "object",
             "description": "Outbound and inbound specification.",
             "properties": {
-                "origin": { "$ref": "#/domains/TrainBooking/types/DateTimeTerminal" },
-                "destination": { "$ref": "#/domains/TrainBooking/types/DateTimeTerminal" }
+                "origin": {"$ref": "#/domains/TrainBooking/types/DateTimeTerminal"},
+                "destination": {"$ref": "#/domains/TrainBooking/types/DateTimeTerminal"},
+                "lowestPrice": {
+                    "type": "object",
+                    "description": "'Start from' price for each class.",
+                    "properties": {
+                        "secondClass": {
+                            "$ref": "#/domains/Generic/types/Price"
+                        },
+                        "firstClass": {
+                            "$ref": "#/domains/Generic/types/Price"
+                        }
+                    },
+                    "required": [
+                        "secondClass"
```

Comment:
is secondClass always available? Can't we just have `lowestPrice` as Price?

Reply:
I think...well maybe....ok, will do the change.
They need lowest price for both classes. And it's the lowest, starting from.